daml-security-test-evidence.csv

```diff
@@ -0,0 +1,939 @@
+testSuiteName,testDescription,property,asset,attacker,threat,mitigation,happyCase,unimplemented,ignored,testPath

```

Comment:
Could this file be generated into a subdirectory (e.g. 'generated/') instead of the repository's root?

Reply:
This file is not to be kept. It was / is just there to be able to inspect the generated CSV file.

Reply:
I rebased and removed all commits labeled with "REMOVE" now.